README.md

```diff
@@ -1,2 +1,70 @@
 # pulumi-package-publisher
 A composite action for publishing Pulumi packages
+
+_This Action is currently in preview and meant for internal use. Any functionality can and likely will change._
+
+### Purpose
+This Action automates setup and publication to the package registries of four Pulumi languages.
+
+### Setup
+
+Set the following environment variables in your Workflow and your GitHub Action secrets:
+
+```yaml
+env:
+    DOTNETVERSION: |
+    6.0.x
+    3.1.301
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    GOVERSION: 1.20.1
+    JAVAVERSION: "11"
+    NODEVERSION: 16.x
+    NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+    NUGET_PUBLISH_KEY: ${{ secrets.NUGET_PUBLISH_KEY }}
+    PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+    PYTHONVERSION: "3.9"
+    SIGNING_KEY: ${{ secrets.JAVA_SIGNING_KEY }}
+    SIGNING_KEY_ID: ${{ secrets.JAVA_SIGNING_KEY_ID }}
+    SIGNING_PASSWORD: ${{ secrets.JAVA_SIGNING_PASSWORD }}
+    SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+    TRAVIS_OS_NAME: linux
```

Comment:
I thought we didn't use travis anymore. Is this still necessary?

Reply:
It's a relic - the [ci script requires it pretty explicitly if I'm reading it right](https://github.com/pulumi/scripts/blob/master/ci/publish-tfgen-package#L29) - I didn't want to spent too much time fixing up the script since we're going to remove it.

Reply:
Thanks for clarifying. I agree we shouldn't try to fix it, since we will remove the script soon. 

Is it possible that we can set it on behalf of our users, instead of asking them to set it?